src/main/java/core/basesyntax/strategy/ReturnOperation.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.Inventory;
+
+public class ReturnOperation implements FruitOperationHandler {
+    @Override
+    public void executeOperation(FruitTransaction transaction, Inventory inventory) {
+        inventory.getInventory().put(transaction.getFruit(),
+                inventory.getInventory().getOrDefault(
+                        transaction.getFruit(), 0) + transaction.getQuantity());
+    }
```

Comment:
Ensure that this strategy is being used correctly in the context of the application. It should be one of the strategies in a map of operation strategies that is passed into a service in the `main()` method, as suggested in the checklist.